src/OPEN_TOOLS_GOWIN.py

```diff
@@ -0,0 +1,436 @@
+import os
```

Comment:
I hadnt even seen this new file yet whoops

Totally fine to start copy-paste like this - but does seem ripe for making some kind of shared version of OPEN_TOOLS with this in it somehow?

Reply:
On this (even though I removed the file from the pull request), I think the shared version of OPEN_TOOLS should work - I was initially testing it and found that the docs were unclear whether the parts I have access to were supported by `nextpnr-gowin` at the moment. It seems like source code says so, but docs say no.